Procfile

```diff
@@ -0,0 +1,2 @@
+release: python manage.py migrate
```

Comment:
😱

Reply:
The expectation was that all the work on configuring the project to deploy on heroku would have a separate PR. Look into how to [**move**](https://stackoverflow.com/questions/2369426/how-to-move-certain-commits-to-another-branch-in-git) the commits with the changes to a separate branch and open a PR.

Reply:
@devGenie ☝️